### PR TITLE
Adds yegor256 award image to README.rst

### PR DESCRIPTION
**Pull request by LadySith**
Created Awards section and added yegor256 award image to README.rst #2924

### CODE REVIEW

**Comment by sils** [comment]
hm, I don't think I'd put this in an own section, rather to the other badges? Also I guess we might want to have it on the website.

File: README.rst [hunk 1]
```diff
@@ -233,6 +233,14 @@ meta information in `setup.py <setup.py>`__ for the current maintainers.
 
 -----
 
+======
+Awards
+======
+
+.. image:: http://www.yegor256.com/images/award/2016/winner-sils.png
```
Comment (AbdealiJK) [inline README.rst]: Can the width be modified to be made a bit smaller ? Seems too clunkt

**Comment by srisankethu** [comment]
The [issue](https://github.com/coala/coala/issues/2924) related to this PR is done. This PR can be closed.

**Comment by sils** [comment]
ouh the issue was never assigned :/

@LadySith especially if you *don't* want to contribute anymore we'd love your opinion on coala.io/newform

**Comment by sils** [comment]
we have to somehow avoid those duplicated PRs better... how can we make the newcomers look at the issue first?

**Pull request closed without merging**